actions/src/main/kotlin/com/alfresco/content/actions/ActionCaptureMedia.kt

```diff
@@ -6,6 +6,7 @@ import com.alfresco.capture.CaptureHelperFragment
 import com.alfresco.content.PermissionFragment
 import com.alfresco.content.data.Entry
 import com.alfresco.content.data.OfflineRepository
+import java.lang.Exception
```

Comment:
This is an unnecessary import the `Exception` reference used here is `com.alfresco.content.actions.Action.Exception`